api/pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>io.druid</groupId>
         <artifactId>druid</artifactId>
-        <version>0.11.0.2</version>
+        <version>0.11.0-lyft-SNAPSHOT</version>
```

Comment:
Since this is repeated , wondering if we can have one property for this version & we can reference this property in the rest of the files ?

Reply:
Ah understood. My only concern was having to update this in the future as it involves changing so many files. I suppose next such change is when we move to 0.12 which wont happen that often. Ok ,lets ship this.